etc/config.xml

```diff
@@ -50,7 +50,7 @@
                 <require_billing_address>0</require_billing_address>
                 <allow_shipping_address_override>1</allow_shipping_address_override>
                 <display_on_shopping_cart>1</display_on_shopping_cart>
-                <order_status>processing</order_status>
+                <order_starttus>processing</order_starttus>
```

Comment:
@mujAhm Please could you chanfge this back to `order_status`

Reply:
@AidanThreadgold ok

Reply:
ah ok I remembered this now